css/layout.css

```diff
@@ -32,3 +44,43 @@
     width: 50%;
   }
 }
+
+.justify-right {
+  justify-content: right;
+}
+
+.justify-left {
+  justify-content: left;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.align-center {
+  align-items: center;
+}
+
+.align-left {
+  align-items: left;
+}
+
+.no-margin {
+  margin: 0;
+}
+
+.no-padding {
+  padding: 0;
+}
+
+.no-padding-left {
+  padding-left: 0;
+}
+
+.no-padding-right {
+  padding-right: 0;
+}
+
+.margin-bottom {
+  margin-bottom: 15px;
+}
```

Comment:
I would specify this with how much margin bottom you gave it. Otherwise it's unclear. Either go with `margin-bottom-15` or `mb-15` for short.